portality/tasks/journal_in_out_doaj.py

```diff
@@ -34,8 +36,8 @@ def run(self):
         job = self.background_job
         params = job.params
 
-        journal_ids = params.get("set_in_doaj__journal_ids")
-        in_doaj = params.get("set_in_doaj__in_doaj")
+        journal_ids = self.get_param(params, "journal_ids")
+        in_doaj = self.get_param(params, "in_doaj")
 
         if journal_ids is None or len(journal_ids) == 0 or in_doaj is None:
```

Comment:
Might be better to factor this out to a `_job_parameter_check` classmethod, see portality.tasks.journal_bulk_edit.JournalBulkEditBackgroundTask for an example definition and use.

The exact same logic needs to be executed both in prepare() and in run() at the beginning of the methods. Putting the definition in one place should make it much harder to modify the condition only in one place, forgetting about the other one. (In fact I've suggested a change to this if condition in the prepare() method below.)